### PR TITLE
Added manifest.json to enable support for LPM.

### PR DESCRIPTION
**Pull request by PerilousBooklet**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "addons": [
+    {
+      "id": "todotreeview",
+      "version": "1.0",
```
Comment (Guldoman) [inline manifest.json]:
```suggestion
      "version": "0.1.1",
```
To continue from the version in the plugins repo.

File: manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      "mod_version": "3",
+      "description": "A TODO tree view for Lite XL",
```
Comment (Guldoman) [inline manifest.json]:
```suggestion
      "description": "Todo tree viewer for annotations in code like `TODO`, `BUG`, `FIX`, `IMPROVEMENT`",
```
Same description used in the plugins repo.

File: manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      "files": [
+        {
+          "url": "https://github.com/drmargarido/TodoTreeView/blob/master/todotreeview.lua"
+        }
+      ]
```
Comment (Guldoman) [inline manifest.json]:
```suggestion
      "path": "todotreeview-xl.lua"
```

**Comment by drmargarido** [comment]
Thanks!

**Comment by Guldoman** [comment]
Opened a PR in the plugins repo to point to the latest commit lite-xl/lite-xl-plugins#329.
Let me know if there's anything else you want to add before 0.1.1 is "released" there.

**Comment by drmargarido** [comment]
Use that commit, I don't have anything new to add.

**Pull request merged**